test/prism/PrismLogicTest.scala

```diff
@@ -330,4 +330,51 @@ class PrismLogicTest extends FreeSpec with Matchers {
       }
     }
   }
-}
\ No newline at end of file
+
+  "sortInstancesByStack" - {
+    "should sort by stack, app, stage" - {
```

Comment:
This nesting doesn't do anything. As-is I'd combine these into a single block, but see what you think after you've read the more lengthy description I've put below.